packages/jaeger-ui/src/components/Monitor/ServicesView/index.tsx

```diff
@@ -86,7 +86,11 @@ const AdaptedVirtualSelect = reduxFormFieldAdapter({
 
 const serviceFormSelector = formValueSelector('serviceForm');
 const oneHourInMilliSeconds = 3600000;
+const oneMinuteInMilliSeconds = 60000;
 export const timeFrameOptions = [
+  { label: 'Last 5 minutes', value: oneMinuteInMilliSeconds * 5 },
+  { label: 'Last 15 minutes', value: oneMinuteInMilliSeconds * 15 },
+  { label: 'Last 30 minutes', value: oneMinuteInMilliSeconds * 30 },
```

Comment:
nit: similar to those below, I would switch the operands to improve readability and consistency. i.e. `5 * oneMinuteInMilliSeconds`, etc.

Reply:
👍🏽